src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/controller/advice/ControllerAdvice.kt

```diff
@@ -23,14 +23,14 @@ class ControllerAdvice {
   @ExceptionHandler(EntityNotFoundException::class)
   @ResponseStatus(HttpStatus.NOT_FOUND)
   fun handle(e: EntityNotFoundException): ResponseEntity<ErrorResponse?> {
-    log.info("EntityNotFoundException: {}", e.message)
+    log.error("EntityNotFoundException: {}", e.message)
```

Comment:
Are we OK loggging at error here in term of support? Will anyone start hassling us if we log errors?

Reply:
No I don't think so,  I was making it consistent with the rest of this file. I'll work out what happens if we handle a 404, I think at the moment we'll log at Warn in the service then throw EntityNotFound and log at Error here,  which is redundant.

Reply:
we don't need both for certain

Reply:
I've removed the warn because we log in the exception handler